index.html

```diff
@@ -38,8 +39,28 @@
   <body>
     <header>
       <a href="#" class="name_logo">Gidiz Logo</a>
+
```

Comment:
- [ ] OPTIONAL: I have noticed that the document has overflow on the right side, but because you set `body: { overflow-x: hidden;}` it is not visible. However, as you can guess it is wrong approach. 
 I recommend to take a look at shapes you added, they are being main reason of the overflow. This requirement is optional and does not stop you from completing the project 😉

Reply:
Thank you for the hint, it helped me resolve he isuue.